config/initializers/secure_headers.rb

```diff
@@ -44,8 +44,13 @@
   config.cookies = {
     secure: true, # mark all cookies as "Secure"
     httponly: true, # mark all cookies as "HttpOnly"
+    # We need to set the SameSite setting to "Lax", not "Strict" due to a bug
+    # in Chrome that resets the session in the new browser tab that opens when
+    # the email confirmation link is clicked. Resetting the session means losing
+    # all the SP info we stored there, meaning during account creation, a user
```

Comment:
Yes that makes sense -- do the easy fix now, potentially re-introduce the more secure `secure headers` setting later alongside a refactor that includes the `request_id` param in more places so we aren't so reliant on the session 🍒